code/modules/gear_presets/survivors/solaris/preset_solaris.dm

```diff
@@ -80,6 +80,46 @@
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots(new_human), WEAR_FEET)
 	..()
 
+/datum/equipment_preset/survivor/solaris_ridge/co_survivor
+	name = "CO Survivor - Solaris Marshal"
+	assignment = "CMB Marshal"
+	job_title = JOB_CMB_TL
+	flags = EQUIPMENT_PRESET_START_OF_ROUND
+	role_comm_title = "CMB MAR"
+	paygrades = list(PAY_SHORT_CMBM = JOB_PLAYTIME_TIER_0)
+	minimap_icon = "deputy"
+	faction_group = list(FACTION_MARSHAL, FACTION_MARINE, FACTION_SURVIVOR)
+	idtype = /obj/item/card/id/marshal
+	skills = /datum/skills/cmb/cosurvivor
+
+/datum/equipment_preset/survivor/solaris_ridge/co_survivor/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/CM_uniform, WEAR_BODY)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/accessory/storage/holster, WEAR_ACCESSORY)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/accessory/holobadge/cord, WEAR_ACCESSORY)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/sunglasses/sechud, WEAR_EYES)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/head/CMB, WEAR_HEAD)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/marine/veteran/cmb/light, WEAR_JACKET)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/marine, WEAR_WAIST)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/backpack/satchel/sec, WEAR_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/gloves/marine/veteran, WEAR_HANDS)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/veteran/pmc/knife, WEAR_FEET)
```

Comment:
/veteran gear is giga doom armor ERT gear, should use different ones for survs rather than having them have poorly distinguished heavy armor on hands and feet.